### ISSUE TITLE
What is the logits and how to get them from kenLM

### ISSUE THREAD

**Issue by ThanhLV97** [kensho-technologies/pyctcdecode#87]

**Comment by qinyuenlp** [comment]
I think the logits matrix comes from your upstream model. It can not be got from kenlm.
